### PR TITLE
readme: fix network deployments link

### PR DESCRIPTION
**Pull request by KailMillor**
https://community.optimism.io/docs/useful-tools/networks/ is not working.

### CODE REVIEW

**Review by ajsutton — COMMENTED** [review]
hmm, I'm not sure that's necessarily the right replacement link. @cpengilly do you know what the best replacement link would be for this?

File: README.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ All other components and packages should be considered development components on
 ### Development branch
 
 The primary development branch is [`develop`](https://github.com/ethereum-optimism/optimism/tree/develop/).
-`develop` contains the most up-to-date software that remains backwards compatible with the latest experimental [network deployments](https://community.optimism.io/docs/useful-tools/networks/).
+`develop` contains the most up-to-date software that remains backwards compatible with the latest experimental [network deployments](https://docs.optimism.io/builders/node-operators/rollup-node#build-your-node).
```
Comment (smartcontracts) [inline README.md]: Correct link is here: https://docs.optimism.io/chain/networks

Comment (KailMillor) [reply README.md]: Thanks!

**Comment by smartcontracts** [comment]
Hi! Could you actually rebase this PR so that it's using the latest code from develop? That will get CI to pass.

**Comment by KailMillor** [comment]
> Hi! Could you actually rebase this PR so that it's using the latest code from develop? That will get CI to pass.

Done, thanks!

**Review by smartcontracts — APPROVED** [review]

**Comment by smartcontracts** [comment]
Merged, thanks!

**Pull request merged**